libs/dh/market-participant/actors/feature-organizations/src/lib/dh-organizations-overview.component.ts

```diff
@@ -116,8 +132,7 @@ export class DhOrganizationsOverviewComponent {
       return;
     }
 
-    const dataToSort = structuredClone<DhOrganization[]>(this.dataSource.filteredData);
-    const dataSorted = this.dataSource.sortData(dataToSort, this.dataSource.sort);
+    const data = structuredClone<Organization[]>(this.dataSource.filteredData);
```

Comment:
question: is the data sorted at this point? The download should reflect the sorting in the table.